runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -1592,6 +1592,10 @@ CISCTransform2FindBytes(TR_CISCTransformer *trans)
       }
    else
       {
+      // the static table currently cannot be relocated
+      if (comp->compileRelocatableCode())
+         return false;
```

Comment:
I think it would be good to trace here like the above early exits do, so that we can see in a log why the loop wasn't transformed. Such tracing should be guarded by `disptrace`, which corresponds to `traceIdiomRecognition`, or at least by some other tracing option (e.g. `TR_TraceOptDetails`). (The above tracing should also be guarded the same way, but that doesn't necessarily need to be modified for this PR)

Reply:
If you want to trace under a more general option like `TR_TraceOptDetails`, I recommend testing your tracing under that option to make sure that it's clear which loop the message is about

Reply:
I will use disptrace